srcpkgs/Orthanc/template

```diff
@@ -0,0 +1,58 @@
+# Template file for 'Orthanc'
+pkgname=Orthanc
+version=1.11.1
+revision=1
+build_wrksrc="OrthancServer"
+build_style=cmake
+configure_args="-DALLOW_DOWNLOADS=ON -DSTATIC_BUILD=OFF -DDCMTK_DIR=/usr/lib/cmake/dcmtk"
+hostmakedepends="python3 unzip tar doxygen dcmtk"
+makedepends="libuuid-devel boost-devel libcurl-devel
+ libdcmtk-devel gtest-devel libpng-devel libjpeg-turbo-devel
+ sqlite-devel libcivetweb-devel openssl-devel jsoncpp-devel lua-devel
+ pugixml-devel"
+depends="dcmtk"
+short_desc="Lightweight RESTful DICOM server"
+maintainer="Carlos E. Gallo F. <gcarlos@disroot.org>"
+license="GPL-3.0-or-later"
+homepage="https://www.orthanc-server.com/index.php"
+distfiles="https://www.orthanc-server.com/downloads/get.php?path=/orthanc/Orthanc-${version}.tar.gz"
+checksum=a634adc9f6dd0a74ebaeb415657b5f3d51822b4fb347641dc44f3bf8b5b56db5
+ignore_elf_dirs="/usr/share/orthanc"
```

Comment:
ignore_elf_dirs is for emulator artifacts and such. Actual executable code like here need to be moved elsewhere, e.g. /usr/lib/orthanc

Reply:
okay, I will move it to /usr/l/ib/orthanc/plugins/ dir